services/core-web/src/components/homepage/HomeTopLinks.tsx

```diff
@@ -45,30 +45,35 @@ const HomeTopLinks = () => {
       route: routes.NOTICE_OF_WORK_APPLICATIONS.route,
       key: "now",
       icon: () => <FontAwesomeIcon icon={faTrianglePersonDigging} />,
+      datacy: "home-link-button-now",
```

Comment:
hmm, considering that these are all in the format of "home-link-button"-${key} I'm thinking we can simplify this by just having the logic performed directly to calculate that instead of passing as a prop.

Reply:
Updated. I removed datacy attributed and changed the replaced the key attribute with id. The reason for doing this is because React treats **key** as a special reserved property.